src/Game/GameFormatterInterface.php

```diff
@@ -0,0 +1,10 @@
+<?php namespace JoeStrong\RetroAchievements\Game;
+
+interface GameFormatterInterface
+{
+    /**
+     * @param Game[] $games
+     * @return string
+     */
+    public function format(array $games) : string;
```

Comment:
Style differs from actual implementation `public function format(array $games) : string` vs `public function format(array $games): string`

Reply:
Good spot. PSR doesn't care and I'm inconsistent. I should configure my IDE better.